transit_odp/static/openapi/fares3.yml

```diff
@@ -71,29 +71,28 @@ paths:
             maxItems: 4
             example: [51.401, 51.509, 0.01, 0.201]
           description: >
-            Limit results to fares data sets that contain information for the area within the rectangular boundingBox you set using co-ordinates</br></br>
+            Limit results to fares data sets that contain information for the area within the rectangular boundingBox you set using co-ordinates.</br></br>
```

Comment:
Bounding box change needs to be made here too.

Reply:
@surabhika Added these in.